tests/unit/rangeslider/rangeslider_core.js

```diff
@@ -139,30 +139,33 @@
 		rangeslider.rangeslider( { disabled: true } );
 
 		ok( !!inputFirst.attr( "disabled" ),  "first input is disabled" );
-		ok( inputFirst.hasClass( "mobile-slider-disabled"),
+		assert.hasClasses( inputFirst, "mobile-slider-disabled",
 			"first input gets mobile-slider-disabled" );
 		ok( !!sliderFirst.attr( "aria-disabled" ), "first slider is aria-disabled" );
-		ok( sliderFirst.hasClass( "ui-state-disabled"), "first slider has ui-state-disabled" );
+		assert.hasClasses( sliderFirst, "ui-state-disabled", 
+			"first slider has ui-state-disabled" );
 		ok( !!inputLast.attr( "disabled" ),  "last input is disabled" );
-		ok( inputLast.hasClass( "mobile-slider-disabled"),
+		assert.hasClasses( inputLast, "mobile-slider-disabled", 
 			"last input gets mobile-slider-disabled" );
 		ok( !!sliderLast.attr( "aria-disabled" ), "last slider is aria-disabled" );
-		ok( sliderLast.hasClass( "ui-state-disabled"), "last slider has ui-state-disabled" );
+		assert.hasClasses( sliderLast, "ui-state-disabled", "last slider has ui-state-disabled" );
 
 		rangeslider.rangeslider( {disabled: false } );
 
 		deepEqual( !!inputFirst.attr( "disabled" ), false, "first input is enabled" );
-		ok( !inputFirst.hasClass( "mobile-slider-disabled"),
-			"first input gets mobile-slider-disabled" );
+		assert.lacksClasses( inputFirst, "mobile-slider-disabled",
+			"first input lacks mobile-slider-disabled" );
 		deepEqual( sliderFirst.attr( "aria-disabled" ), "false",
 			"first slider aria-disabled is false" );
-		ok( !sliderFirst.hasClass( "ui-state-disabled"), "first slider ui-state-disabled removed" );
+		assert.lacksClasses( sliderFirst, "ui-state-disabled",
+			"first slider lacks ui-state-disabled" );
 		deepEqual( !!inputLast.attr( "disabled" ), false, "last input is enabled" );
-		ok( !inputLast.hasClass( "mobile-slider-disabled"),
-			"last input gets mobile-slider-disabled" );
+		assert.lacksClasses( inputLast, "mobile-slider-disabled",
+			"last input lacks mobile-slider-disabled" );
 		deepEqual( sliderLast.attr( "aria-disabled" ), "false",
 			"last slider aria-disabled is false" );
-		ok( !sliderLast.hasClass( "ui-state-disabled"), "last slider ui-state-disabled removed" );
+		assert.lacksClasses( sliderLast, "ui-state-disabled",
+			"last slider lacks ui-state-disabled" );
```

Comment:
I'm not sure that would gain us much since the classes and attributes change for each test we'd still be writing it all out anyway.